docs/source/serving/openai_compatible_server.md

```diff
@@ -4,7 +4,7 @@ vLLM provides an HTTP server that implements OpenAI's [Completions](https://plat
 
 You can start the server using Python, or using [Docker](deploying_with_docker.rst):
 ```bash
-python -m vllm.entrypoints.openai.api_server --model meta-llama/Llama-2-7b-hf --dtype float32 --api-key token-abc123
+python -m vllm.entrypoints.openai.api_server --model mistralai/Mistral-7B-Instruct-v0.2 --dtype auto --api-key token-abc123
```

Comment:
nit: Let's remove `--api-key`?

Reply:
I suggest to keep the api key arg. Its a low overhead way to ensure basics of access control, especially given the prive of GPU. Having the api key in the QuickStart promotes security best practices (so many quick start configs end up being the production ones..) let's make this one right :)